pandas/core/indexes/multi.py

```diff
@@ -990,15 +990,11 @@ def _constructor(self):
     def _shallow_copy(self, values=None, **kwargs):
         if values is not None:
             names = kwargs.pop("names", kwargs.pop("name", self.names))
-            # discards freq
-            kwargs.pop("freq", None)
```

Comment:
This makes no sense, ``freq`` isn't accepted by any MultiIndex constructor.

Reply:
+1